packages/webapp/src/pages/election/round-video-upload.tsx

```diff
@@ -0,0 +1,348 @@
+import React, { useEffect, useState } from "react";
+import { RiVideoUploadLine } from "react-icons/ri";
+import { Dayjs } from "dayjs";
+
+import {
+    FluidLayout,
+    isValidDelegate,
+    onError,
+    uploadIpfsFileWithTransaction,
+    uploadToIpfs,
+    useCommunityGlobals,
+    useCurrentElection,
+    useLocalStorage,
+    useOngoingElectionData,
+    useUALAccount,
+} from "_app";
+import {
+    Container,
+    Text,
+    Expander,
+    Heading,
+    Loader,
+    VideoSubmissionFormAndPreview,
+    VideoSubmissionPhase,
+    Button,
+    Link,
+} from "_app/ui";
+import {
+    CurrentElection,
+    ElectionStatus,
+    ErrorLoadingElection,
+    setElectionRoundVideo,
+    getRoundTimes,
+} from "elections";
+import { EncryptionPasswordAlert } from "encryption";
+import { EdenMember } from "members";
+import { RoundHeader } from "elections/components/ongoing-election-components";
+
+// TODO:
+// 1) add buttons that launch video upload service in new tab
+// 2) test inductions to ensure I haven't screwed up their video upload
+// 3) Test video uploads during (and after) an election
+
+export const RoundVideoUploadPage = () => {
+    const [isElectionComplete, setIsElectionComplete] = useState(false);
+    const {
+        data: currentElection,
+        isLoading: isLoadingElection,
+        isError: isErrorElection,
+    } = useCurrentElection();
+
+    useEffect(() => {
+        if (currentElection?.electionState === ElectionStatus.Registration) {
+            setIsElectionComplete(true);
+        }
+    }, [currentElection]);
+
+    const isLoading = isLoadingElection;
+    const isError = isErrorElection;
+
+    if (isLoading || !currentElection) return <LoaderSection />;
+    if (isError) return <ErrorLoadingElection />;
+
+    const renderBanner = true; // TODO: get end time for video submission
```

Comment:
question: @cool-ant what is the idea of this unused `renderBanner` variable?  cc @brandonfancher, maybe you know?

Reply:
I believe it was consumed in the banner prop on the `<FluidLayout />` before you replaced that with `<SideNavLayout />`.

Reply:
ok! removed!